src/store/consts/utils.ts

```diff
@@ -253,6 +254,27 @@ export const calculateTimeRemaining = (slot: number, nextRoundStartSlot: BN): BN
   }
   return slotDiff.muln(slotTime)
 }
+export const getAddressFromIndex = (
+  synthetics: { [x: string]: ISynthetic },
+  collateral: { [x: string]: ICollateral },
+  nr: number
+) => {
+  let syntheticAddress: string = ''
```

Comment:
there wouldn't be a problem when there are multiple syntetics/collaterals with the same asset index? currently we have two collaterals for which this happens

Reply:
Right, I added the correct condition